packages/xchain-binance/__tests__/client.test.ts

```diff
@@ -65,9 +65,9 @@ describe('BinanceClient Test', () => {
   const testnetaddress_path1 = 'tbnb1vjlcrl5d9t8sexzajsr57taqmxf6jpmgaacvmz'
 
   const singleTxFee = baseAmount(37500)
-  const transferFee = { type: 'base', average: singleTxFee, fast: singleTxFee, fastest: singleTxFee }
+  const transferFee = { type: 'base' as FeeType, average: singleTxFee, fast: singleTxFee, fastest: singleTxFee }
```

Comment:
Pls don't use `as` type casting ... It's just a sign types don't not work properly.